docs/02-connect/02-connect_your_wallet.md

```diff
@@ -37,14 +37,22 @@ Adding a network example:
 
 ![alt_text](/img/docs/get-started/metamask-add-network2.png)
 
+##### EON Mainnet
 
+      Network name:  EON Mainnet
+      New RPC URL:  https://eon-rpc.horizenlabs.io/ethv1
+      Chain ID: 7332
+      Currency symbol: ZEN
+      Block Explorer: https://eon-explorer.horizenlabs.io/
+
+Once you have specified the RPC endpoint and Chain ID to make a connection, a notification appears stating, **“EON Mainnet was successfully added”**.
 
 ##### Gobi Testnet
 
       Network name:  Gobi Testnet
       New RPC URL:  https://gobi-testnet.horizenlabs.io/ethv1
       Chain ID: 1663 
-      Currency symbol: ZEN
+      Currency symbol: tZEN

```

Comment:
@jmontesdeocanuez @yurikoinaba can we confirm if the testnet currency should be `tZEN` or `TZEN`? The EON [dashboard](https://eon.horizen.io/status/gobi) and most vars I've seen uses `TZEN`

Reply:
Let's keep it consistent w other placements in the documentation

Reply:
<img width="275" alt="image" src="https://github.com/HorizenOfficial/evm-documentation/assets/36055057/070cb6c4-16af-44a5-a35d-03b75f01aad3">
<img width="243" alt="image" src="https://github.com/HorizenOfficial/evm-documentation/assets/36055057/f9ccee2d-0edc-4ed7-8b5e-d9d5d42b2685">

Actually, right now we're not consistent, we are using both of 'tZEN' and 'TZEN' across the site. Should I update all to be the same? In that case, should them be 'tZEN' or 'TZEN' @yurikoinaba

Reply:
@jmontesdeocanuez Ok in that case, let's make it `TZEN` everywhere. Thank you both